### DISCUSSION TITLE
MasterHamsterKombatBot Logging

### DISCUSSION THREAD

**Discussion by tboy1337** [masterking32/MasterHamsterKombatBot#301] [Show and tell]
Local logging code for MasterHamsterKombatBot, at the beginning of each week (Monday) the bot will zip the old log file and create a new one to reduce the file size by approximately 90%, the log file location is Logs/output.log.  Also the zipped logs get their name changed to the date and time they are being zipped so there isn’t any conflict with new zipped logs, tested fully working.

If anyone wants to use it replace main.py with [main.py](https://github.com/tboy1337/MasterHamsterKombatBot/blob/test/main.py) in my test branch.
Previous discussion thread https://github.com/masterking32/MasterHamsterKombatBot/discussions/147.

**Comment by masterking32** [comment]
Stop creating issue/discussion about this shit.  I never add this to the project.
